src/main/java/com/radixdlt/crypto/HashHandler.java

```diff
@@ -1,20 +1,20 @@
-/*
- * (C) Copyright 2020 Radix DLT Ltd
- *
- * Radix DLT Ltd licenses this file to you under the Apache License,
- * Version 2.0 (the "License"); you may not use this file except in
- * compliance with the License.  You may obtain a copy of the
- * License at
- *
- *  http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
- * either express or implied.  See the License for the specific
- * language governing permissions and limitations under the License.
- */
-
+/*
+ * (C) Copyright 2020 Radix DLT Ltd
+ *
+ * Radix DLT Ltd licenses this file to you under the Apache License,

```

Comment:
Incorrect license format, similar to above except here it's a missing whitespace (what?)

Reply:
yeah strange, I must have messed something up, found in 16 places, found and replaced :D